lib/picotls.c

```diff
@@ -1630,8 +1630,9 @@ static int setup_traffic_protection(ptls_t *tls, int is_enc, const char *secret_
 
     ctx->epoch = epoch;
 
-    log_secret(tls, log_labels[ptls_is_server(tls) == is_enc][epoch],
-               ptls_iovec_init(ctx->secret, tls->key_schedule->hashes[0].algo->digest_size));
+    if (tls->key_schedule)
```

Comment:
```suggestion
    if (tls->key_schedule != NULL)
```
For consistency with https://github.com/h2o/h2o/wiki/Coding-Style#if.